pkg/reconciler/testing/v1beta1/listers.go

```diff
@@ -180,6 +182,10 @@ func (l *Listers) GetApiServerSourceV1alpha2Lister() sourcev1alpha2listers.ApiSe
 	return sourcev1alpha2listers.NewApiServerSourceLister(l.indexerFor(&sourcesv1alpha2.ApiServerSource{}))
 }
 
+func (l *Listers) GetApiServerSourceV1beta1Lister() sourcev1beta1listers.ApiServerSourceLister {
```

Comment:
Should this be added to pkg/testing/listers.go as well?

Reply:
I created another follow up issue here: https://github.com/knative/eventing/issues/3629
I think ideally we should only have one lister.go under pkg/testing. But I'm not sure why the codebase has two lister.go under pkg/testing. I added it here because that's what v1alpha2 ApiSeverSource uses. I think for container source, you can use add it in either one now. Later, we can clean it up for all resources in https://github.com/knative/eventing/issues/3629

Reply:
Yeah that makes sense. I was wondering the same thing. Container source test uses testing/listers.go so I'll add it there for now. But thanks for creating the issue!

Reply:
I took another look at the codebase. I think pkg/testing/v1 is for v1 resources, pkg/testing/v1beta1 is for v1beta1 resources. I think it's correct that I put lister under  pkg/testing/v1beta1 is correct. 
But unlike broker, sources use different code style. They don't create resources under specific version package.
I think we need another PR to clean it up